src/types/itemTypes.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
+// Item structure coming from the Search Results API response
+export interface SearchResultsItem {
+  "@id": string
+  idNyplSourceId?: ItemSourceID
+  accessMessage?: ItemAccessMessage[]
+  shelfMark?: string[]
+  electronicLocator?: string[]
```

Comment:
electronicLocator has been deprecated, so I think it's safe to remove it. We used to return items from the discovery api with electronic resources mixed in, so electronicLocator was where the urls were stored. we don't do that anymore. electronic resources are attached to the bib only.